backend/src/v2/compiler/argocompiler/argo.go

```diff
@@ -28,6 +29,7 @@ import (
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/structpb"
 	k8score "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
the same import is just above this one :)

Reply:
Done, removed this import and accordingly also updated the remaining code to use the `k8score` import instead.